Tasks/ServiceFabricDeploy/ServiceFabricSDK/Publish-UpgradedServiceFabricApplication.ps1

```diff
@@ -200,9 +203,21 @@ function Publish-UpgradedServiceFabricApplication
 
         $reg = Get-ServiceFabricApplicationType -ApplicationTypeName $names.ApplicationTypeName | Where-Object  { $_.ApplicationTypeVersion -eq $names.ApplicationTypeVersion }
         if ($reg)
-        {
-            Write-Host (Get-VstsLocString -Key SFSDK_UnregisteringExistingAppType -ArgumentList @($names.ApplicationTypeName, $names.ApplicationTypeVersion))
-            $reg | Unregister-ServiceFabricApplicationType -Force
+        {            
+            try {
+                Write-Host (Get-VstsLocString -Key SFSDK_UnregisteringExistingAppType -ArgumentList @($names.ApplicationTypeName, $names.ApplicationTypeVersion))            
+                $reg | Unregister-ServiceFabricApplicationType -Force    
+            }
+            catch [System.Fabric.FabricException] {
+                ## If SkipUpgrade param is set when existing version already deployed, then back out of upgrade
+                if($SkipUpgradeSameTypeAndVersion -And $Error[0].Exception.Message -like '*Application type and version is still in use*') {                    
```

Comment:
@NCarlsonMSFT Changed the logic from being exception handling to pre-validation using the existing code that checked for existing of app in the cluster. Was actually much simpler than the first implementation.

But seems like my new VS Code settings removed a lot trailing whitespace from the file. Should I revert and re-apply my changed to keep PR to only new features or do you prefer the removal of trailing whitespace?